invenio_vocabularies/contrib/subjects/jsonschemas/subjects/subject-v1.0.0.json

```diff
@@ -1,21 +1,28 @@
 {
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "local://subjects/subject-v1.0.0.json",
   "description": "Subjects from specific vocabularies.",
   "properties": {
-    "identifier": {
-      "description": "Subjects term identifier.",
+    "$schema": {
+      "$ref": "local://definitions-v1.0.0.json#/$schema"
+    },
+    "id": {
+      "description": "Subject unique identifier.",
       "type": "string"
     },
-    "scheme": {
-      "description": "Identifier scheme.",
+    "name": {
```

Comment:
data mode is tbd, but should we call this `subject`? UI is ready to configure this field name. Name makes sense too